src/client-config.js

```diff
@@ -12,7 +12,7 @@ window.config = {
   API_GATEWAY_URL: 'http://localhost:7070/',
   CONFIG_API_URL: 'http://localhost:2021',
   LOGIN_URL: 'http://localhost:3020',
-  AUTH_URL: 'http://localhost:4040',
+  AUTH_URL: 'http://localhost:7070/auth',
```

Comment:
I was thinking if we should just get rid of these configuration values altogether but maybe it makes sense to keep them here? This would enable pointing calls made by client

`/refreshToken`
`/invalidateToken`

to country config for instance if more custom logic is wanted. 

We need to consolidate all auth endpoints at some point and verify we are following some standard schema

Reply:
Hmm, I guess we could get rid of them, and then allow configuring which auth_url gateway then eventually forwards the requests to. Getting to configure it here allows having a auth service which bypasses gateway altogether but not sure if there are use cases for that 🤔